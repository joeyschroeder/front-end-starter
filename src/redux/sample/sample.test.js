import { DEFAULT_STATE, resetSampleValue, sampleReducer, updateSampleValue } from './sample';

describe('sampleReducer', () => {
  describe('updateSampleValue', () => {
    it('should add 1 to state.sample.count', () => {
      const action = updateSampleValue();
      const result = sampleReducer(DEFAULT_STATE, action);
      expect(result).toMatchSnapshot();
    });
  });

  describe('resetSampleValue', () => {
    it('should reset to state.sample.count to 0', () => {
      const action = resetSampleValue();
      const result = sampleReducer(DEFAULT_STATE, action);
      expect(result).toMatchSnapshot();
    });
  });
});
