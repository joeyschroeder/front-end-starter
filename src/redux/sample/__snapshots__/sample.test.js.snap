// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sampleReducer resetSampleValue should reset to state.sample.count to 0 1`] = `
Object {
  "sampleValue": 0,
}
`;

exports[`sampleReducer updateSampleValue should add 1 to state.sample.count 1`] = `
Object {
  "sampleValue": 1,
}
`;
